api_types.ts

```diff
@@ -546,6 +546,20 @@ export interface CommonPackFormulaDef<T extends ParamDefs> {
    * @hidden
    */
   readonly allowedAuthenticationNames?: string[];
+
+  /**
+   * The JavaScript function that implements parameter validation
+   *
+   * This function takes in parameters and a context containing permissions mode information,
+   * and validates the parameters.
+   *
+   * @returns true if the parameters are valid
+   * @throws {@link ParameterValidationError} if the parameters are invalid
+   */
+  validateParameters?(
+    params: ParamValues<ParamDefs>,
+    context: ExecutionContext | SyncExecutionContext,
+  ): Promise<true>;
```

Comment:
Opinion: Only being able to return `true` is a bit unexpected. I'd either expect `boolean` or `void`.

Reply:
Okay sounds good. Then I prefer void where returning without an error thrown means valid.